### ISSUE TITLE
NUSD v1.9 - Crash on selecting "Emulate Wii System Update"

### ISSUE THREAD

**Issue by GoogleCodeExporter** [WB3000/nusdownloader#13]
```
What steps will reproduce the problem?
1. Open NUSD v1.9
2. Select "Scripts"
3. Select "Emulate Wii System Update"
3. Select my region "Europe"

What is the expected output? What do you see instead?
Crash of the application, Popup with the description of the error 
(System.Xml.XmlException, line 1, position 1, wrong data)

What version of the product are you using? On what operating system?
- NUSD v1.9
- Windows 7 Pro x64 / Windows XP Pro x86

Please provide any additional information below.

```

Original issue reported on code.google.com by `nicola.s...@yahoo.it` on 1 Mar 2011 at 1:48
- Merged into: #12

**Comment by GoogleCodeExporter** [comment]
```
Please see issue12 !
```

Original comment by `cs-nut...@live.fr` on 1 Mar 2011 at 3:28

**Comment by GoogleCodeExporter** [comment]
```
I've seen issue12 but there is no solution or reply from the authors
```

Original comment by `nicola.s...@yahoo.it` on 1 Mar 2011 at 9:20

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
I think this problem is caused by a change of Wii Update url.
```

Original comment by `rappersk...@gmail.com` on 13 Mar 2011 at 3:11

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
No, I can't but mayvbe the author can.
```

Original comment by `rappersk...@gmail.com` on 14 Mar 2011 at 12:45

**Comment by GoogleCodeExporter** [comment]
```
Merged into #12
```

Original comment by `gb.luke` on 19 Mar 2011 at 6:08
- Changed state: **Duplicate**
